test/TensorFlowNET.UnitTest/NativeAPI/Eager/CApi.Eager.Execute_MatMul_CPU.cs

```diff
@@ -19,14 +20,18 @@ public unsafe void Execute_MatMul_CPU()
         unsafe void Execute_MatMul_CPU(bool async)
         {
             using var status = TF_NewStatus();
-            var opts = TFE_NewContextOptions();
-            c_api.TFE_ContextOptionsSetAsync(opts, Convert.ToByte(async));
+
+            static SafeContextHandle NewContext(bool async, SafeStatusHandle status)
+            {
+                using var opts = c_api.TFE_NewContextOptions();
+                c_api.TFE_ContextOptionsSetAsync(opts, Convert.ToByte(async));
+                return c_api.TFE_NewContext(opts, status);
+            }
 
             IntPtr t;
-            using (var ctx = TFE_NewContext(opts, status))
+            using (var ctx = NewContext(async, status))
```

Comment:
⚠️ This pull request intentionally uses two different patterns for updating test code. The first pattern, seen here, uses a local function for the creation of the context. This is a bigger change (in terms of lines of code), but allows `using` statements to be used for both `opts` and `ctx` without changing the lifetime of `opts`.

❓ Which of the two patterns do you prefer? I will update this pull request to use the same pattern for all cases.

Reply:
I like local function style. It looks more structrual.

Reply:
Great! I'll get this updated tomorrow, and when I push to this it will automatically reduce the review to only the new part now that #559 is merged.

Reply:
➡️ Now updated to this form.